underfs/s3/src/main/java/tachyon/underfs/s3/S3UnderFileSystem.java

```diff
@@ -371,6 +374,10 @@ private StorageObject getObjectDetails(String key) {
    * @return the the parent key
    */
   private String getParentKey(String key) {
+    // Root does not have a parent.
```

Comment:
NIT: Usually, arranging multiple `if-else` in the following pattern makes src code much easier to follow (for a new/average audience):
```
if (foo) {
  return false;
}
if (bar) {
  return false;
}
return SomeFunc();
```